### ISSUE TITLE
Automatic Services

### ISSUE THREAD

**Issue by SimonAronsson** [ItsHale/Hale-CheckPack-Core#2]
Ability to monitor automatic services, and a pre-written powershell script to restart them when they go down.

**Comment by piksel** [comment]
"automatic" as in startup-type?

**Comment by SimonAronsson** [comment]
Correct. In the case of nodes being used as app-servers. A common way to monitor the behavior of black box app servers is to just make sure that the automatic services are up and running.
